Jenkinsfile

```diff
@@ -157,6 +157,7 @@ pipeline {
                         deleteDir()
                         unstash 'MathSetup'
                         sh "echo CXX=${MPICXX} >> make/local"
+                        sh "echo CXX_TYPE=gcc >> make/local"                        
```

Comment:
Why do we need to specifically define the `CXX_TYPE` here? Isnt this handled by the make/compiler_flags code?

Reply:
No. This is special for mpi. With mpi the compiler is called mpicxx or something like that. The our internal detection fails for that such that you need to tell Stan what is the wrapped compiler type.

Reply:
Ah, OK. Did not know that.